tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "module": "CommonJS",
-    "target": "ES2019",
+    "module": "ES2020",
```

Comment:
This drops support for everyone using CJS on their projects. I'm in favor of it, CJS is outdated and a anchor dragging us, but just making sure this is a an intended decision.

It might still work for most cases, but TS won't be guaranteeing that so seemingly unrelated changes could suddenly downstream CJS consumers.

Reply:
We need at least ES2020 to preserve the `import` statement, otherwise, it will be compiled into `require`, and won't work as intended when bundled.

Reply:
From my experience, it usually works out of the box, as modern setups support both CJS and ES6 imports. The only exception that typically requires manual configuration is Jest. Do you have any other concerns here?